peering/peering_test.go

```diff
@@ -0,0 +1,6 @@
+package peering
+
+import "testing"
+
+func TestPeeringService(t *testing.T) {
```

Comment:
😢

Reply:
Oops. I went and wrote a sharness test instead. But you're right, I should have one here as well.

Reply:
Done.